fixOldSyntax.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/golem-client/build.rs b/golem-client/build.rs
+index e93bafa..16eb4c1 100644
+--- a/golem-client/build.rs
++++ b/golem-client/build.rs
+@@ -27,8 +27,8 @@ fn main() {
+         )
+         .unwrap();
+ 
+-        println!("cargo::rerun-if-changed=build.rs");
```

Comment:
This file tells me that this nix package is trying to build Golem with an older rust version (the `::` syntax is since 1.77). We don't want to pin our codebase to an old rust version, and are potentially using new rust features so even though we could change this particular syntax back but the build can break any time if it's not using the same rust version that we are using (which is, always the latest, for now).

Reply:
I can try to run an update (remove/update flake.lock) and update the dependencies we might wont need this line anymore.